tests/mock/CMakeLists.txt

```diff
@@ -13,14 +13,16 @@ set(MOCK_MOC_HEADERS
   MockServer.h
 )
 
+qt5_wrap_cpp(MOC_OUTFILES ${MOCK_MOC_HEADERS})
```

Comment:
MOC step should already be handled by common_library! I don't know why but I couldn't use common_library for this lib in the past, but good if it works now :-)